generator/src/lib.rs

```diff
@@ -85,13 +85,30 @@ named!(cfloat<&str, f32>,
     terminated!(nom::float_s, char!('f'))
 );
 
+macro_rules! khronos_link (
+    ($name:expr) => {
+        Literal::string(&format!("<https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/{name}.html>", name=$name))
+    }
+);
```

Comment:
Why is this a macro?

Reply:
It started out as something more complicated. I also thought it might need to be embedded into vk.rs like some of the others, but that didn't turn out to be needed. Reducing to a function should be doable.

Reply:
Now a function